gazebo/igvc_description/urdf/jessi.urdf

```diff
@@ -331,7 +333,7 @@
   <joint name="imu_to_front_ball" type="floating">
     <parent link="imu"/>
     <child link="front_ball"/>
-    <origin xyz=".39 0 -.53" rpy="0 0 0" />
+    <origin xyz=".39 -0 -.53" rpy="0 0 0" />
```

Comment:
There shouldn't be a negative sign in front of the middle entry (`-0`) in the value of the `xyz` attribute